internal/errors/option.go

```diff
@@ -105,3 +105,48 @@ func (e *ErrCriticalOption) Error() string {
 func (e *ErrCriticalOption) Unwrap() error {
 	return e.origin
 }
+
+// ErrUnsetOption represents the unset option error.
+type ErrUnsetOption struct {
```

Comment:
ErrIgnoredOption sounds more understandable

Reply:
@kpango I think Ignore means it is ignored, and will not process anything, so I think ErrUnsetOption is better.
What do you think?

Reply:
@kevindiu but for IndexPath it seems ignored...

Reply:
How about ErrUnconfiguredOption?
I think the Unset is usually misunderstood as the meaning of the Getter/Setter pattern, which reverts the set to Empty, so I think it's better to use other words like Ignored or Unconfigured.

Reply:
I start thinking that ErrIgnoreOption is reasonable that it is ignored and debug log will be printed.
@rinx @hlts2 @vankichi what do you think?

Reply:
🤔

Reply:
IMO, `ErrIgnoredOption` is meaningful

Reply:
@vankichi sounds great.

@kevindiu can you please fix it?

Reply:
@kpango I have fixed this, thank you for your comment :)

Reply:
> what do you think?
ok,  sounds good.!
thank you